internal/config/config.go

```diff
@@ -471,19 +550,25 @@ func (c *Config) RemoveProviderExcept(provider cloudprovider.Provider) {
 // was put inside an image just by looking at its name.
 func (c *Config) IsDebugImage() bool {
 	switch {
-	case c.Provider.GCP != nil:
-		gcpRegex := regexp.MustCompile(`^projects\/constellation-images\/global\/images\/constellation-v[\d]+-[\d]+-[\d]+$`)
-		return !gcpRegex.MatchString(c.Provider.GCP.Image)
+	case c.Provider.AWS != nil:
+		// TODO: Add proper image name validation for AWS when we are closer to release.
+		return true
 	case c.Provider.Azure != nil:
 		azureRegex := regexp.MustCompile(`^\/CommunityGalleries\/ConstellationCVM-b3782fa0-0df7-4f2f-963e-fc7fc42663df\/Images\/constellation\/Versions\/[\d]+.[\d]+.[\d]+$`)
 		return !azureRegex.MatchString(c.Provider.Azure.Image)
+	case c.Provider.GCP != nil:
+		gcpRegex := regexp.MustCompile(`^projects\/constellation-images\/global\/images\/constellation-v[\d]+-[\d]+-[\d]+$`)
```

Comment:
regexp with `MustCompile` must be global var, not in a func. (not your code, I know)

Reply:
Moved it 👍🏻